src/game.py

```
@@ -141,13 +134,33 @@ def generate_random_hint(self):
                 airport.events.append(Event( random.choice(hint_events), {"survivor":0}))
 
+    def check_win(self):
+        if self.player.location.is_safe:
+            # Win case
+            display_win_screen( self.player)
+            self.end_game(has_won=True)
+            return True
+        else:
+            return False
+    def check_lose(self):
+        if self.player.fuel <= 0 and not self.has_won and self.player.inventory.items["fuel"] == 0 and self.player.location.is_explored == True or self.player.health <= 0 and not self.has_won:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if self.player.location.is_explored:` for truth checks

Replace with `self.player.location.is_explored`

(E712)